netneurotools/plotting.py

```diff
@@ -490,7 +492,8 @@ def plot_fsaverage(data, *, lhannot, rhannot, order='LR', surf='pial',
         # finally, add data to this hemisphere!
         brain.add_data(vtx_data, vmin, vmax, hemi=hemi, mid=center,
                        thresh=thresh + 0.5, alpha=1.0, remove_existing=False,
-                       colormap=colormap, colorbar=colorbar, verbose=False)
+                       colormap=colormap, colorbar=colorbar, verbose=False,
+                       **data_kws)
```

Comment:
If people don't supply `data_kws` and it defaults to `None` then this call will fail—you can't unpack a `NoneType` like this. To resolve this you'll need to add a check in the function body that does something like:

```python
if data_kws is None:
    data_kws = {}
```

(Don't make the default parameter a dictionary—having mutable objects as parameter defaults is considered bad practice.)